corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js

```diff
@@ -13,6 +13,14 @@ hqDefine('app_manager/js/case-config-ui-advanced.js', function () {
 
     var CaseConfig = function (params) {
         var self = this;
+        self.help_name = gettext('Case Property Description');
+        self.default_description = gettext("You do not have a description for this case property. If you would like to add one you can do so in the data dictionary.");
```

Comment:
The truncations shouldn't be a problem; the inline edit widget lets you specify a class for the read-only view. Yeah, offline with me if you give it a shot and want a hand.